### PR TITLE
Kibana Script Added with log

### PR DESCRIPTION
**Pull request by abhaysingh1722**
[kibana-6.4.2-2018-10-11-10%3A51%3A00.log](https://github.com/imdurgadas/scripts/files/2468652/kibana-6.4.2-2018-10-11-10.3A51.3A00.log)

### CODE REVIEW

File: Kibana/install.sh [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+function checkPrequisites() {
+	if command -v "sudo" >/dev/null; then
+		printf -- 'Sudo : Yes\n'
+	else
+		printf -- 'Sudo : No \n'
+		printf -- 'You can install the same from installing sudo from repository using apt, yum or zypper based on your distro. \n'
+		exit 1
+	fi
+}
```
Comment (imdurgadas) [inline Kibana/install.sh]: Interactive shell prompting for Confirmation is missing. Please check cAdvisor.
In confirmation message, you will have to highlight all the dependencies you are installing or upgrading like node js etc.

File: Kibana/install.sh [hunk 2]
```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+# © Copyright IBM Corporation 2017, 2018.
+# LICENSE: Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+
+set -e
+
+PACKAGE_NAME="kibana"
+PACKAGE_VERSION="6.4.2"
+#OVERRIDE=false
```
Comment (imdurgadas) [inline Kibana/install.sh]: Replace `OVERRIDE` with `FORCE`

File: Kibana/install.sh [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+	sudo zypper install wget tar >/dev/null
+	configureAndInstall
+	;;
+
+*)
+	printf -- "%s not supported \n" "$DISTRO" | tee -a "${LOG_FILE}"
+	exit 1
+	;;
+esac
+
```
Comment (imdurgadas) [inline Kibana/install.sh]: Steps apart from dependencies from repository same for UB,SLES,RH ?

Comment (abhaysingh1722) [reply Kibana/install.sh]: Yes

**Comment by abhaysingh1722** [comment]
[kibana-6.4.2-2018-10-12-04%3A41%3A21.log](https://github.com/imdurgadas/scripts/files/2471836/kibana-6.4.2-2018-10-12-04.3A41.3A21.log)
Verified on Ubuntu 18.04

**Comment by abhaysingh1722** [comment]
Verified on all the distros.
[rh73.log](https://github.com/imdurgadas/scripts/files/2472784/rh73.log)
[rh74.log](https://github.com/imdurgadas/scripts/files/2472785/rh74.log)
[rh75.log](https://github.com/imdurgadas/scripts/files/2472786/rh75.log)
[sles12ps3.log](https://github.com/imdurgadas/scripts/files/2472787/sles12ps3.log)
[sles15.log](https://github.com/imdurgadas/scripts/files/2472788/sles15.log)

**Pull request merged**